packages/xrpl/src/models/ledger/Offer.ts

```diff
@@ -2,6 +2,17 @@ import { Amount } from '../common'
 
 import { BaseLedgerEntry, HasPreviousTxnID } from './BaseLedgerEntry'
 
+export interface Book {
```

Comment:
```suggestion
export interface AdditionalBook {
```

Reply:
I'm basing my changes off of [this file](https://github.com/XRPLF/rippled/pull/5404/files#diff-c4c9724bcf26e225b395e4218f52b07c123b1be58870d836bbe8b731f15d4cef). 

I couldn't find any reference to `sfAdditionalBook` in the rippled codebase.

Here is the `offer` ledger object produced from the `ledger_entry` RPC.

```
{
        Account: 'rMn7X1zJS8pv4R5APWLCrEkVPaSEqj8QPp',
        AdditionalBooks: [ { Book: [Object] } ],
        BookDirectory: '580F976574AC2FDB82181AA0EA5D8C6D7075D7E6F7A2D53053038D7EA4C68000',
        BookNode: '0',
        DomainID: '8BB4C6D27F469EDB167ECD684725DF4E09BE8E158958F5197B3A80AE053C5A65',
        // other fields below
```

`AdditionalBooks: [ { Book: [Object] } ],` is the observed output